testing/simple/src/main/scala/org/example2/Baz.scala

```diff
@@ -4,8 +4,8 @@ package org.example2
 
 class Baz {
   class Wibble {
-    class baz
-    object baz
+    class Baz
```

Comment:
I'm not sure my example really reproduces the problem properly. The ticket has a better example using scalatest inner classes.